src/derive/stages/attributes.rs

```diff
@@ -323,8 +323,8 @@ impl TryFrom<Log> for UserDeposited {
             .into_bytes()
             .unwrap();
 
-        let from = Address::try_from(log.topics[1])?;
-        let to = Address::try_from(log.topics[2])?;
+        let from = Address::from(log.topics[1]);
+        let to = Address::from(log.topics[2]);
```

Comment:
why this change, is this guaranteed to succeed

Reply:
yes see https://github.com/a16z/magi/actions/runs/7011622482/job/19074567685